packages/mongo-nav/src/Loading.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { LogoMark } from '@leafygreen-ui/logo';
+import { css } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const orgNavContainer = css`
+  height: 60px;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 15px;
+  padding-right: 15px;
+  font-size: 13px;
+  line-height: 15px;
+  color: ${uiColors.gray.dark3};
+  border-bottom: 1px solid ${uiColors.gray.light2};
+`;
+
+const projNavContainer = css`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding-left: 15px;
+  padding-right: 15px;
+  height: 45px;
+  box-shadow: 0 3px 7px 0 rgba(67, 117, 151, 0.08);
+  overflow: hidden;
+`;
+
+export default function Loading() {
```

Comment:
Figured it'd be worth it to wait for now rather than adding messy conditional logic throughout OrgNav and ProjectNav (at least for now) but once we have the designs depending on how much we're omitting/still rendering we can figure out what makes the most sense (bringing this into its own component or not)

Reply:
👍